greenswitch/esl.py

```diff
@@ -344,16 +344,31 @@ def call_command(self, app_name, app_args=None):
         # We're not allowed to send more commands.
         # lingering True means we already received a hangup from the caller
         # and any commands sent at this time to the session will fail
```

Comment:
remove the old comment, there not more lingering

Reply:
It seems that it was moved to `_perform_call_command`.

Reply:
You fellas are helps <3